substrate/executor/src/sandbox.rs

```diff
@@ -113,26 +111,22 @@ impl ImportResolver for Imports {
 
 	fn resolve_global(
 		&self,
-		module_name: &str,
-		field_name: &str,
+		_module_name: &str,
+		_field_name: &str,
 		_global_type: &::wasmi::GlobalDescriptor,
 	) -> Result<::wasmi::GlobalRef, ::wasmi::Error> {
-		Err(::wasmi::Error::Instantiation(format!(
-			"Export {}:{} not found",
-			module_name, field_name
-		)))
+		// TODO:
+		unimplemented!()
```

Comment:
returning an error is strictly better than panicking in this and `resolve_table` -- why were these implementations removed?